packages/nouns-webapp/src/App.tsx

```diff
@@ -131,6 +132,7 @@ function App() {
               <Route exact path="/lilnouners" component={NounersPage} />
```

Comment:
It would be nice to rip out the PreLaunch stuff since it'll never be "pre launch" anymore. Not your code or really relevant to this PR but I'm noticing it in here.